File: problems/1491_average-salary-excluding-the-minimum-and-maximum-salary/__main__.py
from typing import List


class Solution:
    def average(self, salary: List[int]) -> float:
        min = salary[0]
        max = salary[0]
        sum = 0
        for s in salary:
            sum += s
            if s < min:
                min = s
            if s > max:
                max = s
        return (sum - min - max) / (len(salary) - 2)

File: problems/1491_average-salary-excluding-the-minimum-and-maximum-salary/main.go
package main

func average(salary []int) float64 {
	min, max, sum := salary[0], salary[0], 0
	for _, s := range salary {
		sum += s
		if s < min {
			min = s
		}
		if s > max {
			max = s
		}
	}
	return float64(sum-min-max) / float64(len(salary)-2)
}
